OSC/ondemand
# Support editing number fields

This is the first step in editing fields properly in the UI. In draft now because builds off of another PR that isn't merged yet.

To test
* create a script with `bc_num_hours` in it.
* Press `Edit` on that form field
* Now you'll see the menu pop open as shown below.
* Pressing `Save` (the save button in the image) will collapse this menu.

![image](https://user-images.githubusercontent.com/4874123/231851116-a75b9d06-2f32-484e-b65f-496ec7b203cc.png)

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1202821104799596/1204393413252743) by [Unito](https://www.unito.io)
